app/core/builder/blueprint.py

```diff
@@ -12,6 +13,28 @@
 
 from app.config import config, ConfigurationError
 from app.core.components.optimizer.dummy import DummyOptimizer
+from mock.resource import ResourceMock
+from mock.environment import Env
```

Comment:
Unsure why we need these here? Isn't the point to load the configs we need? We're going to throw away the Env mock, at least.

Reply:
Same as the other options blueprints serve a way to change sources in runtime. Instead of using the default config and have to restart the service